Rubberduck.Core/UI/Controls/DependencyObjectExtensions.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Windows;
+using System.Windows.Media;
+
+namespace Rubberduck.UI.Controls
+{
+    public static class DependencyObjectExtensions
+    {
+        //from https://stackoverflow.com/a/41985834/1188513
```

Comment:
I think it'll be useful to have more explanation on why / how the class is used inline rather than just link. You know how SO frowns on link-only answer. 😉

Reply:
Gone, wasn't used anymore.